pyproject.toml

```diff
@@ -53,6 +53,9 @@ numpy = "^1.19.0"
 grpcio = "^1.27.2"
 google = "^2.0.3"
 protobuf = "^3.12.1"
+types-protobuf = "^3.17.4"
+types-setuptools = "^57.0.2"
+types-dataclasses = "^0.1.7"
```

Comment:
Can these also go to the section `[tool.poetry.dev-dependencies]` or are they required to be in the production dependencies?

Reply:
Moved it to `dev`